sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/KeyEncryptionKeyClientBuilder.java

```diff
@@ -136,50 +114,6 @@ public KeyEncryptionKey buildKeyEncryptionKey(KeyVaultKey key) {
         return Mono.defer(() -> Mono.just(new KeyEncryptionKeyAsyncClient(keyId, pipeline, serviceVersion)));
     }
 
-    /**
-     * Creates a {@link KeyEncryptionKeyAsyncClient} based on options set in the builder.
-     * Every time {@code buildAsyncKeyEncryptionKey(String)} is called, a new instance of
-     * {@link KeyEncryptionKeyAsyncClient} is created.
-     *
-     * <p>If {@link KeyEncryptionKeyClientBuilder#pipeline(HttpPipeline) pipeline} is set, then the {@code pipeline}
-     * and {@code keyId} are used to create the {@link KeyEncryptionKeyAsyncClient async client}.
-     * All other builder settings are ignored. If {@code pipeline} is not set, then
-     * ({@link KeyEncryptionKeyClientBuilder#credential(TokenCredential) jsonWebKey vault credential} and
-     * {@code keyId} are required to build the {@link KeyEncryptionKeyAsyncClient async client}.</p>
-     *
-     * @param key the key to be used for crypto operations.
-     *
-     * @return A {@link KeyEncryptionKeyAsyncClient} with the options set from the builder.
-     * @throws IllegalStateException If {@link KeyEncryptionKeyClientBuilder#credential(TokenCredential)} or
-     * {@code key} have not been set.
-     */
-    public Mono<? extends AsyncKeyEncryptionKey> buildAsyncKeyEncryptionKey(KeyVaultKey key) {
```

Comment:
This is a breaking change. These public methods should not be removed from the builder.

Reply:
This was introduced in beta